package.json

```
@@ -14,21 +17,27 @@
     "statuses": "^2.0.1"
   },
   "devDependencies": {
+    "@arethetypeswrong/cli": "^0.17.4",
+    "@tsconfig/node18": "^18.2.4",
+    "@types/node": "^22.13.10",
```

```suggestion
    "@types/node": "^18.11.9",
```
@RobinTail Should we use the @types/node version of the first major we support?

---

I believe it does not matter, because here it's declared within `devDependencies` so that `@types/node` is used internally, @Phillip9587 
The consumer would have their own version of Node.js types.